cmd/eskip/writeclient.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import "github.com/zalando/skipper/eskip"
+import (
+	"errors"
+	etcdclient "github.com/zalando/skipper/etcd"
```

Comment:
i think these aliases are very useful here. They are longer, they don't solve any conflict, and just make searching in the code harder.